maven-plugin-annotations/src/main/java/org/apache/maven/plugins/annotations/ResolutionScope.java

```diff
@@ -19,12 +19,13 @@
  * under the License.
  */
 
-import org.apache.maven.artifact.Artifact;
-
 /**
  * Dependencies resolution scopes available before
  * <a href="/ref/current/maven-core/apidocs/org/apache/maven/lifecycle/internal/MojoExecutor.html">mojo execution</a>.
  *
+ * Important note: The {@code id} values of this enum corresponds to constants of
+ * {@code org.apache.maven.artifact.Artifact} class and MUST BE KEPT IN SYNC (not that they changed recently).
```

Comment:
delete "(not that they changed recently)"

Reply:
fixed